backend/server/src/db/migrations/6-blocks.sql

```diff
@@ -22,8 +22,10 @@ create table blocks (
     -- Text blocks
     text_content text,
 
-    -- Video list blocks
-    videolist_series bigint references series on delete restrict,
+    -- Series blocks
+    series_id bigint references series on delete restrict,
+
+    -- All videolist-like blocks
     videolist_layout video_list_layout,
     videolist_order video_list_order
```

Comment:
Should we want to rename these as well? :thinking:

Reply:
Mh no, my idea was that there will be other "video list like" blocks that will also use these variables. Like "the most popular videos of this realm" or sth like that: this should essentially look the same as the series block and also have the layout/order options.

Reply:
Aah okay, that makes sense.